akvo/rsr/management/commands/report_d2d.py

```diff
@@ -6,82 +6,146 @@
 
 
 import datetime
+import tablib
 
 from django.contrib.admin.models import LogEntry
 from django.core.management.base import BaseCommand
 from django.db.models import Q, Count
 
 from akvo.rsr.models import (
     IatiExport, Organisation, Project, IndicatorPeriodData,
-    IndicatorPeriodDataComment,
+    IndicatorPeriodDataComment, LoginLog, PublishingStatus,
+    ProjectUpdate
 )
 
 
-def projects_using_results_framework(before_date):
-    """Return projects which had activity in the results framework before a date."""
+def get_active_organisations(date_start, date_end):
+    has_project_updates = set(
+        ProjectUpdate.objects
+        .filter(Q(created_at__range=(date_start, date_end)) | Q(last_modified_at__range=(date_start, date_end)))
+        .values_list('project__id', flat=True)
+    )
+    has_indicator_updates = set(
+        IndicatorPeriodData.objects
+        .filter(Q(created_at__range=(date_start, date_end)) | Q(last_modified_at__range=(date_start, date_end)))
+        .values_list('period__indicator__result__project__id', flat=True)
+    )
+    pids = has_project_updates | has_indicator_updates
+    return set(Project.objects.filter(id__in=pids).values_list('primary_organisation__id', flat=True))
+
```

Comment:
This makes 3 calls to the DB instead of a single one. It should return a `QuerySet`